### ISSUE TITLE
How's the speed for inference time ?

### ISSUE THREAD

**Issue by Stephenfang51** [HRNet/DEKR#1]
Excellent Work !
just wonder any benchmark for inferece time ?

Thanks !

**Comment by josemusso** [comment]
Hi!. On a 1660ti max-q gives around 2.7 fps. On a Tesla T4 around 3 fps. I'm figuring out how to reduce inference time.

**Comment by xsacha** [comment]
Getting 10fps without flip and 7fps with flip on a Titan V.

Haven't been able to trace the model and preserve size. Also, torchvision op (deform conv2d) is an issue when trying to run it in c++.

**Comment by josemusso** [comment]
> Getting 10fps without flip and 7fps with flip on a Titan V.
> 
> Haven't been able to trace the model and preserve size. Also, torchvision op (deform conv2d) is an issue when trying to run it in c++.
> 
> I saved a lot of time by removing a lot of the post-processing of the posemodel and heatmap. There's lots of work there and all it is doing is adding the x, y coordinate to every value in the posemodel.

Hi @xsacha ! Could you give me some detail on the removed post processing? or could you share your code? I m now starting tests on a RTX 3060, could give you some performance details later on.

**Comment by ZXin0305** [comment]
> Getting 10fps without flip and 7fps with flip on a Titan V.
> 
> Haven't been able to trace the model and preserve size. Also, torchvision op (deform conv2d) is an issue when trying to run it in c++.
> 
> I saved a lot of time by removing a lot of the post-processing of the posemodel and heatmap. There's lots of work there and all it is doing is adding the x, y coordinate to every value in the posemodel.

hello! Have you reduced the inference time again??
It is also a low speed about 10 fps when apply the project in reality.
Thanks!

**Comment by xsacha** [comment]
Yeah I have given up on this one sorry since the inference time was too high and the deformable conv makes it harder to use. Dropping the deform conv made the accuracy lower.

**Comment by Deep-learning999** [comment]
hrnet is very cumbersome, is there a lightweight model that can be applied to this algorithm

**Comment by lucasjinreal** [comment]
that's too slow... with this speed I can apply a more accurate top-down model with higher accuracy....

**Comment by wmcnally** [comment]
@jinfagang check out KAPAO, it’s pretty fast.

**Comment by xsacha** [comment]
@wmcnally Thanks. That looks quite fast. The videos look very inaccurate (stolen joints) but I guess they are showing the small model. It can be resolved as this is usually post-process.
